frontend/test/metabase/scenarios/alert/email_alert.cy.spec.js

```diff
@@ -25,14 +25,22 @@ describe("scenarios > alert > email_alert", () => {
     });
   });
 
-  describe.skip("alert set up", () => {
+  describe("alert set up", () => {
     // NOTE: To run tests, first run `python -m smtpd -n -c DebuggingServer localhost:1025` in your terminal
     beforeEach(() => {
       cy.server();
 
       cy.visit("/admin/settings/email");
       cy.findByText("SMTP Host");
-      setupEmail();
+      // *** setupEmail();
+      signInAsAdmin();
+      cy.request("PUT", "/api/email", {
+        settings: {
+          "email-smtp-host": "localhost",
+          "email-smtp-port": "1025",
+          "email-from-address": "test@local.host",
```

Comment:
```suggestion
          "email-from-address": "test@metabase.test",
```
It's better to use `.test` domains instead of TLDs that actually exists - just in case we some day start sending mails to those addresses:
https://en.wikipedia.org/wiki/Top-level_domain#Reserved_domains

Reply:
Thanks for the heads up!